shared-libs/transitions/test/unit/lib/utils.js

```diff
@@ -4,13 +4,19 @@ const registrationUtils = require('@medic/registration-utils');
 const taskUtils = require('@medic/task-utils');
 const config = require('../../../src/config');
 const db = require('../../../src/db');
+require('chai').should();
```

Comment:
The existing tests in this file are using the `should` functionality.  It technically works when all the tests are run together since a previous test file initialized the `should`, but when I just ran this file by itself, it failed without this line...